Jint/Runtime/Debugger/BreakLocation.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Jint.Runtime.Debugger
+{
+    /// <summary>
+    /// BreakLocation is a combination of an Esprima position (line and column) and a source (path or identifier of script).
+    /// Like Esprima, first column is 0 and first line is 1.
+    /// </summary>
+    public sealed class BreakLocation : IEquatable<BreakLocation>
```

Comment:
you could also start using records here, would remove a lot of boiler-plate code

Reply:
Done 🙂